docs/source/api/user-guide/transformations/aggregation-quantile.ipynb

```diff
@@ -293,7 +293,7 @@
             "source": [
                 "## Quantile via B-ary Tree\n",
                 "\n",
-                "A slightly more complicated algorithm that tends to provide better utility is to privatize a B-ary tree instead of a histogram.\n",
+                "A slightly more complicated algorithm that tends to provide better utility is to sanitize a B-ary tree instead of a histogram.\n",
```

Comment:
```suggestion
                "A slightly more complicated algorithm that tends to provide better utility is to release a DP B-ary tree instead of a histogram.\n",
```